### PR TITLE
Control the test timeout values from test.properties #3934

### LINKED ISSUE #3934: Control the test timeout values from test.properties
Not only does this standardize and get rid of those magic numbers, this also allows us to control the timeout value as we wish according to CPU power/memory availability.

### PR DESCRIPTION
**Pull request by wkurniawan07**
Fixes #3934 
I tried running the full test suite and it passes without any problem (sadly not straight green), the timing also remains more or less the same despite the longer timeout (15 seconds standardized).

### CODE REVIEW

File: src/test/resources/test.template.properties [hunk 1]
```diff
@@ -36,6 +36,12 @@ test.selenium.host=localhost
 test.firefox.path=
 
 ###############################################################################
+# This is the timeout value for the waitings done in browsers
+# e.g waiting for element (non-)presence/(in)visibility, for page to load
+# This value can be adjusted depending on your CPU power/memory availability.
+test.timeout=15
```
Comment (damithc) [inline src/test/resources/test.template.properties]: In what units?

Comment (wkurniawan07) [inline src/test/resources/test.template.properties]: Oh, right. In seconds.

File: src/test/java/teammates/test/pageobjects/StudentProfilePage.java [hunk 2]
```diff
@@ -79,7 +79,7 @@ public void fillProfilePic(String fileName) throws Exception {
 
     public void showPictureEditor() throws Exception {
         uploadPopupButton.click();
-        waitForElementToBecomeVisible("studentPhotoUploader");
+        waitForElementVisible(browser.driver.findElement(By.id("studentPhotoUploader")));
```
Comment (damithc) [inline src/test/java/teammates/test/pageobjects/StudentProfilePage.java]: `waitForElementVisible` doesn't sound right. `waitForElementVisibility`?

**Comment by wkurniawan07** [comment]
@bamboo3250 if you don't mind :p

**Comment by bamboo3250** [comment]
Sure, no problem xD

**Comment by bamboo3250** [comment]
Good to merge. Just remember to tell others update their `test.properties` after merging.

**Pull request merged**